galleon-pack/galleon-shared/src/main/resources/layers/standalone/microprofile-reactive-messaging-kafka/layer-spec.xml

```diff
@@ -10,6 +10,7 @@
         <prop name="org.wildfly.rule.properties-file-match-mp-kafka-property" value="[/META-INF/microprofile-config.properties,/WEB-INF/classes/META-INF/microprofile-config.properties],mp.messaging.connector.smallrye-kafka.*"/>
         <prop name="org.wildfly.rule.properties-file-match-mp-kafka-outgoing" value="[/META-INF/microprofile-config.properties,/WEB-INF/classes/META-INF/microprofile-config.properties],mp.messaging.outgoing.*.connector,smallrye-kafka"/>
         <prop name="org.wildfly.rule.properties-file-match-mp-kafka-incoming" value="[/META-INF/microprofile-config.properties,/WEB-INF/classes/META-INF/microprofile-config.properties],mp.messaging.incoming.*.connector,smallrye-kafka"/>
+        <prop name="org.wildfly.rule.add-on-description" value="Support for MicroProfile Reactive Messaging Kafka connector."/>
```

Comment:
@kabir 

"connectors"?  Or is it just 1 in which case it would be "Support for a MicroProfile..." (added 'a').

Reply:
It is singular. At present we only have one MP RM connector, i.e. Kafka. But AMQP is in the works #16281 and would be its own add-on.

I would say "Support for the..." rather than 'a' ;-)

Reply:
Thank-you. Fixed.